app/models/user.rb

```diff
@@ -155,8 +155,12 @@ def send_vote_reminder_email
     UserMailer.reminder_to_vote(self).deliver_now
   end
 
+  def name
+    self[:name].try { |n| n + test_user_suffix }
+  end
```

Comment:
I feel a bit uncomfortable about interfering with what ActiveRecord says about the name ... but I can see why you did it. The alternative would mean choosing another method name to represent the concept (`name_for_ui` perhaps ? ... a bit yuk) and then changing callers where appropriate. maybe later, on a refactoring ?

Reply:
Agreed on all points! I dunno haha